docs/courses/csintro3/orientation/math.md

```diff
@@ -1,101 +1,171 @@
 
-```typescript
-let num: number = 5 + 3 * 2;
-console.log("" + num);
+As mentioned above, when using blocks, we needed to use a separate block for each Math operator. In JavaScript, it is a lot easier to string these operators together.
+
+For example, what was 
+
+```blocks
+let num = 0
+num = 1 + (2 + 3)
 ```
 
+is now
+
 ```typescript
-let num: number = 15 / 3 * 8;
+let num: number = 1 + 2 + 3
 console.log("" + num);
 ```
 
-## Assignment Operators
-
-### Show increment / change by transition (blocks -> JavaScript)
+Some other examples are:
 
 ```typescript
-let num: number = 0;
-num = num + 5;
-num = num + 3;
+let num: number = 5 + 3 * 2;
+console.log("" + num);
 ```
 
 ```typescript
-let num: number = 0;
-num += 5;
-num += 3;
+let num: number = 24 / 3 * 4;
+console.log("" + num);
 ```
 
-### Special case: add or subtract 1 (increment / decrement operators)
+## Order of Operations
+
+Think about how these might be ambiguous. What value do these expressions evaluate to? It depends based on which operation you do first. As it turns out, JavaScipt uses a **PEMDAS** structure to determine which operations should be done in what order.
+
+|Order	|Operation 		|
+|:---:	|:---			|
+|1st	|Parenthesis	|
+|2nd	|Exponents		|
+|3rd	|Multiplication	|
+|		|Division		|
+|4th	|Addition	 	|
+|		|Subtraction 	|
+
+Operations that are of the same order (e.g. multiplication and division) will be completed left-to-right. That is ```24 / 3 * 4``` evaluates to 32 because 24 is divided by 3 before the multiplication happens.
+
+While parenthesis are not an operator, they serve as a way to group operations together. Operations done within parenthesis will be completed first before moving onto those outside.
+
+If we look at the example
 
 ```typescript
-let num: number = 0;
-num += 1;
-num += 1;
-num += -1;
-num += 1;
+let num: number = 24 / 3 * 4;
+console.log("" + num)
 ```
 
+This will print out the number 32 because the division occurs before the multiplication. If we want to change this order and multiply before we divide, we can use parenthesis like
+
 ```typescript
-let num: number = 0;
-num++;
-num++;
-num--;
-num++;
+let num: number = 24 / (3 * 4)
+console.log("" + num)
 ```
 
-Explain why -> very commonly used, easier to write as a short hand, etc
+This will output 2 instead
 
-## Other assignment operators (quick over view as less commonly used at first)
 
-### minus equals
+## Student Task 1:
 
-```typescript
-let num: number = 15;
-num = num - 5;
-```
+1. Pick 4 random numbers and write them horizontally on piece of paper
+2. In the 3 spaces between those numbers, place a random operator to put there
+3. Evaluate the expression on paper
+4. Type out the expression in JavaScript like the above examples and compute the result
+
+## Assignment Operators
+
+There are many instances in which you may want to set a variable to an expression that references itself, such as
 
 ```typescript
-let num: number = 15;
-num -= 5;
+num = num + 5;
```

Comment:
either have to declare this as invalid ts (\`\`\`typescript-ignore) or declare the variable first